Source/NSApplication.m

```diff
@@ -2496,30 +2496,37 @@ - (void) hide: (id)sender
 	      [_hidden_main resignMainWindow];
 	    }
 	  
-	  windows_list = GSOrderedWindows();
-	  iter = [windows_list reverseObjectEnumerator];
+          // Sends -hidewindow: to the main menu. If window manager supports
+          // _GNUSTEP_WM_HIDE_APP atom - call succeeds - our windows will be
```

Comment:
As guy is agnostic about the different backend implementations we should not be talking about specific X atoms here. Rather whether the backend handles the hiding (via the window manager) or we need to do it here.

Reply:
This comment is just explain logic. Do I need to remove comment?

Reply:
Rewrite not remove :-)
This actually is not what the window backend will do and also not what the x11 backend does for most window managers.

Reply:
Fixed.